frigate/video.py

```diff
@@ -377,6 +378,7 @@ def receiveSignal(signalNumber, frame):
         objects_to_track,
         object_filters,
         detection_enabled,
+        improve_contrast_enabled,
```

Comment:
This also needs to be removed now. I think this should be last change.

Reply:
Done. Thanks for your patience with me!